src/pages/index.js

```diff
@@ -180,8 +240,12 @@ function handleAddCardFormSubmit(inputValues) {
     .then((data) => {
       const cardEl = renderCard(data);
       section.addItem(cardEl);
-      addCardModal.resetForm();
+      /*addCardModal.resetForm();
       addFormValidator.disableButton();
+      addCardModal.close();*/
+
+      addCardFormValidator._disableButton();

```

Comment:
`_disableButton`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.

And you need to do that inside `FormValidator` as well